crispy_forms/tests/test_layout.py

```diff
@@ -606,6 +607,34 @@ def test_use_custom_control_is_used():
     assert response.content.count(b"custom-checkbox") == 0
 
 
+@only_bootstrap4
```

Comment:
Could this benefit from `@pytest.mark.parametrize`? 

https://docs.pytest.org/en/stable/parametrize.html